csrc/type.h

```diff
@@ -341,7 +341,11 @@ enum class BinaryOpType {
   // is boolean op. These ops also don't work on floating point inputs.
   And,
   Or,
-  Xor
+  Xor,
+
+  // generate complex from real and imaginary parts
+  ComplexFloat,
+  ComplexDouble
```

Comment:
Do we need two binary op type? PyTorch only has a single complex https://pytorch.org/docs/stable/generated/torch.complex.html?highlight=complex#torch.complex

Reply:
changed to one.